ts/src/index.ts

```diff
@@ -29,6 +29,7 @@ import {Profiler} from './profiler';
 
 const common: Common = require('@google-cloud/common');
 const pjson = require('../../package.json');
+const heapProfiler = require('bindings')('sampling_heap_profiler');
```

Comment:
It doesn't seem right to call directly into the binding here. The profilers/heap-profiler should be the only level of abstraction on top of that, so call that.

BTW, heap-profiler still calls enable() from its constructor? Doesn't seem right if so - not sure which effects calling that twice has.

Reply:
I've switched to creating a HeapProfiler, which is passed in to the profiler.